backend/controllers/iepHistory.js

```diff
@@ -38,13 +39,35 @@ const getIepHistoryFactory = elite2Api => {
 
     iepHistoryDetails.shift()
 
+    let filteredResults = iepHistoryDetails
```

Comment:
Definitely, personal preference but might be cleaner to have a filtering logic in its own function, this might remove the need for the let keyword.

Reply:
My preference too, have done so :)